roles/secretary.md

```diff
@@ -26,12 +26,12 @@ Attendance spreadsheet is here: [link forthcoming]()
 
 Board repository is here: [https://github.com/swcarpentry/board/](https://github.com/swcarpentry/board/)
 
-Mentoring committee materials are here [https://github.com/swcarpentry/board/tree/master/subcommittees/mentoring](https://github.com/swcarpentry/board/tree/master/subcommittees/mentoring)
+Mentoring committee documents are located in [Carpentries organisation](https://github.com/carpentries) in its own [repository](/)
 
 ### Submitting meeting minutes
 
 - [ ] After each meeting, star the etherpad. You will find this option on the top
    right of the etherpad.
 - [ ] Convert meeting minutes to a markdown file named `minutes-YYYY-MM-DD.md`,
  saved in `board > subcommittee > mentoring > minutes`.  
```

Comment:
This should be changed to: 
 `mentoring > minutes`

Reply:
fixed in 516f1b2d6bd4d732ae17a34cd2ed8f3028e71f3b